### ISSUE TITLE
Rotated picture apparently cannot be fixed

### ISSUE THREAD

**Issue by Archilegt** [NaturalHistoryMuseum/scratchpads2#6617]
A picture that I see in upright position in my computer shows rotated when uploaded to Scratchpads. See:
https://myriatrix.myspecies.info/content/mhng-arto-22372
https://myriatrix.myspecies.info/file/28582

What could be causing this issue? Some metadata in the image? How to rotate it to the upright position in Scratchpads?

**Comment by therobyouknow** [comment]
I think rotation is a metadata in the image exif data.  If this is the case, then in plain English what this means is that the image orientation i.e. whether it is portrait, landscape, rotated 90 degrees etc. is stored in the settings within the image file. 

Therefore, it may be worth a try editing that exif data of the image before uploading to see if the image is oriented as required.

- https://superuser.com/questions/670818/how-to-automatically-rotate-images-based-on-exif-data
- https://www.makeuseof.com/windows-11-view-edit-exif/

See if this works for you.  

A further option could be for me to add config on your site for a new Drupal media style or adjust existing to handle the rotation.

The rotation setting stored as metadata means that the entire image itself does not need to be processed to be rotated, such processing on lossy formats resulting in introduction of artifacts and image not being identical to original pixel-by-pixel.

**Comment by Archilegt** [comment]
The metadata in the images seems correct. Portrait and landscape may actually be the issue, but it could be a Drupal/Scratchpads issue. How? Because the images which appear rotated do so not because of some orientation metadata but because they are higher than wide. Drupal may assume that the default orientation of images is landscape. Therefore, when reading the height and width of an image, Drupal may have an automatic setting to display it as landscape by default. That will cause "high" images to display rotated.
We would need to find that setting and turn it off. 

@edwbaker worked on this before:
Baker E (2013) EXIF Custom: Automatic image metadata extraction for Scratchpads and Drupal. _Biodiversity Data Journal_, 1: e973. https://doi.org/10.3897/BDJ.1.e973
so he may be aware of landscape/portrait rotation issues and how to solve them.

**Comment by Archilegt** [comment]
Possible solution, independent of the root cause: [Simple Image Rotate](https://www.drupal.org/project/simple_image_rotate)

**Comment by Archilegt** [comment]
This issue could be renamed to "Scratchpads not reading EXIF orientation metadata".
